.stylelintrc.js

```diff
@@ -1,3 +1,6 @@
 module.exports = {
-  extends: 'stylelint-config-ship-shape'
-};
\ No newline at end of file
+  extends: 'stylelint-config-ship-shape',
+  // Handle lint violations as warnings.
+  // Otherwise build fails
+  defaultSeverity: 'warning'
```

Comment:
The build should fail for lint errors, we do not want this.

Reply:
Hi @rwwagner90, feel free to remove. No need to include.